packages/shorebird_cli/lib/src/commands/patch/ios_patcher.dart

```diff
@@ -334,15 +343,10 @@ class IosPatcher extends Patcher {
         dumpDebugInfoPath: dumpDebugInfoDir?.path,
       );
 
-      if (dumpDebugInfoDir != null) {
-        final debugInfoZip = await dumpDebugInfoDir.zipToTempFile();
-        debugInfoZip.copySync(p.join('build', debugInfoFile.path));
-        logger.detail(
-          'Link debug info saved to ${debugInfoFile.path}',
-        );
-      }
+      await consolidateDebugInfo();
```

Comment:
can we move this to a `finally` block since we want it to happen regardless of success or failure?